bin/runtime/src/lib.rs

```diff
@@ -140,16 +129,24 @@ pub fn native_version() -> NativeVersion {
     }
 }
 
-const NORMAL_DISPATCH_RATIO: Perbill = Perbill::from_percent(75);
+pub const MILLISECS_PER_BLOCK: u64 = DEFAULT_MILLISECS_PER_BLOCK;
+pub const BLOCKS_PER_HOUR: u32 = 60 * 60 * 1000 / (MILLISECS_PER_BLOCK as u32);
+
+pub const MILLI: Balance = TOKEN / 1000;
+pub const MICRO: Balance = MILLI / 1000;
+pub const NANO: Balance = MICRO / 1000;
+pub const PICO: Balance = NANO / 1000;
+
+// 75% block weight is dedicated to normal extrinsics
+pub const NORMAL_DISPATCH_RATIO: Perbill = Perbill::from_percent(75);
 // The whole process for a single block should take 1s, of which 400ms is for creation,
 // 200ms for propagation and 400ms for validation. Hence the block weight should be within 400ms.
-const MAX_BLOCK_WEIGHT: Weight = 400 * WEIGHT_PER_MILLIS;
+pub const MAX_BLOCK_WEIGHT: Weight = 400 * WEIGHT_PER_MILLIS;
 // We agreed to 5MB as the block size limit.
 pub const MAX_BLOCK_SIZE: u32 = 5 * 1024 * 1024;
 
-pub const MILLICENTS: Balance = 100_000_000;
-pub const CENTS: Balance = 1_000 * MILLICENTS; // 10^12 is one token, which for now is worth $0.1
-pub const DOLLARS: Balance = 100 * CENTS; // 10_000_000_000
+// The storage deposit is roughly 1 TOKEN per 1kB
+pub const DEPOSIT_PER_BYTE: Balance = MILLI;
```

Comment:
I'm not so sure, but e.g. in multisignatures it is just a cost (not refundable), so maybe just `BYTE_COST`/`STORAGE_BYTE_COST` is better?

Reply:
If it's not refundable then why is it called "deposit"? The code seems to be using `T::Currency::reserve` and `T::Currency::unreserve` (https://github.com/paritytech/substrate/blob/bc7a1e6c19aec92bfa247d8ca68ec63e07061032/frame/multisig/src/lib.rs#L711) so I'd naively guess this is a refundable deposit?

Reply:
okay, then I wrongly remembered